syntax/vue.vim

```diff
@@ -50,6 +50,12 @@ syntax include @JS syntax/javascript.vim
 unlet! b:current_syntax
 syntax region javascript keepend matchgroup=Delimiter start=/<script\( lang="babel"\)\?\( type="text\/babel"\)\?>/ end="</script>" contains=@JS fold
 
+syntax include @TS syntax/typescript.vim
+if exists("b:current_syntax")
+  unlet b:current_syntax
+endif
+syntax region typescript keepend matchgroup=Delimiter start=/<script\( ts\| lang="ts"\| lang="typescript"\)>/ end="</script>" contains=@TS fold
```

Comment:
I think it's better to allow any attribute with something like ` \_[^>]*` as in all other script syntaxes